dom-manipulation/main.js

```diff
@@ -0,0 +1,24 @@
+let clickCounter = 0;
+const hotButton = document.querySelector('.hot-button');
+
+const clickCount = document.querySelector('.click-count');
```

Comment:
Don't forget the `$` before your variable names! You should be adding that `$` anytime you store a DOM element in a variable